qms-tests/sanity/tests/documents/REQ-05.spec.ts

```diff
@@ -0,0 +1,104 @@
+    await allure.tms('TESTS-381', 'https://tracex.hc.engineering/workbench/platform/tracker/TESTS-381')
+    const leftSideMenuPage = new LeftSideMenuPage(page)
+    const folderName = generateId(5)
+
+    await leftSideMenuPage.clickButtonOnTheLeft('Documents')
+    await test.step('2. Create a new workspace', async () => {
+      const documentContentPage = new DocumentContentPage(page)
+      await documentContentPage.clickAddFolderButton()
+      await documentContentPage.fillDocumentSpaceForm(folderName)
+      await documentContentPage.clickLeaveFolder(folderName)
+    })
+
+    await attachScreenshot('TESTS-381_workspace_created.png', page)
+  })
+
+  test('TESTS-382. Create new a new Effective Template with category External', async ({ page }) => {
+    await allure.description('Requirement\nUsers need to create a new template')
+    await allure.tms('TESTS-382', 'https://tracex.hc.engineering/workbench/platform/tracker/TESTS-382')
+    const leftSideMenuPage = new LeftSideMenuPage(page)
+    const category = faker.word.words(2)
+    const description = faker.lorem.sentence(1)
+    const code = faker.word.words(2)
+    const title = faker.word.words(2)
+
+    await leftSideMenuPage.clickButtonOnTheLeft('Documents')
+    await test.step('2. Create a new category', async () => {
+      const documentContentPage = new DocumentContentPage(page)
+      await documentContentPage.selectControlDocumentSubcategory('Categories')
+      await documentContentPage.clickOnAddCategoryButton()
+      await documentContentPage.fillCategoryForm(category, description, code)
+      await documentContentPage.clickNewDocumentArrow()
+      await documentContentPage.clickNewTemplate()
+      await createTemplateStep(page, title, description, category)
+    })
+
+    await attachScreenshot('TESTS-382_Template_created.png', page)
+  })
+
+  test('TESTS-383. authorized User can search a doc per category "External"', async ({ page }) => {
+    await allure.description('Requirement\nUsers need to create a new category and space')
+    await allure.tms('TESTS-383', 'https://tracex.hc.engineering/workbench/platform/tracker/TESTS-383')
+
+    const title = faker.word.words(2)
+    const description = faker.lorem.sentence(1)
+    const code = faker.word.words(2)
+    const leftSideMenuPage = new LeftSideMenuPage(page)
+    const category = faker.word.words(2)
+
+    await leftSideMenuPage.clickButtonOnTheLeft('Documents')
+    await test.step('2. Create a new category and template', async () => {
+      const documentContentPage = new DocumentContentPage(page)
+      await documentContentPage.selectControlDocumentSubcategory('Categories')
+      await documentContentPage.clickOnAddCategoryButton()
+      await documentContentPage.fillCategoryForm(category, description, code)
+      await documentContentPage.clickNewDocumentArrow()
+      await documentContentPage.clickNewTemplate()
+      await createTemplateStep(page, title, description, category)
```

Comment:
Here as well. Should it be a standalone step?

Reply:
ok, will mix it a bit